### ISSUE TITLE
[Feature]: export/backup Monal data, from/to others monal plateforms or others xmpp clients? 

### ISSUE THREAD

**Issue by lm2lm2** [monal-im/Monal#1102]
### Precondition

- [X] I understood that this template is only for feature requests and not for bug reports

### Cross-checked related issues?

- [X] I have cross-checked this overview https://github.com/monal-im/Monal/issues/322 as well as filtered for related labels https://github.com/monal-im/Monal/labels

### Describe your feature

The idea is to being able to export/import, account per account, data from/to different Monal clients regarding device type, or even from others xmpp clients..

**Comment by tmolitor-stud-tu** [comment]
Monal to Monal --> Use iCloud to backup and restore the App from one device to another
Other app to/from Monal: hardly possible without data loss, every app has its own database format etc.

One could, however, define a new format containing all the basic things like history, [XEP-0184](https://xmpp.org/extensions/xep-0184.html), [XEP-0333](https://xmpp.org/extensions/xep-333.html), [XEP-0363](https://xmpp.org/extensions/xep-0363.html) etc., just like [XEP-0227](https://xmpp.org/extensions/xep-0227.html) does for servers and write that up in a new XEP.

Then implement that new XEP in commonly used clients like Conversations, Gajim or Monal.

That's nothing I will do, but I'm happy to take pull requests.

**Comment by lm2lm2** [comment]
> Monal to Monal --> Use iCloud to backup and restore the App from one device to another Other app to/from Monal: hardly possible without data loss, every app has its own database format etc.
> 
> One could, however, define a new format containing all the basic things like history, [XEP-0184](https://xmpp.org/extensions/xep-0184.html), [XEP-0333](https://xmpp.org/extensions/xep-333.html), [XEP-0363](https://xmpp.org/extensions/xep-0363.html) etc., just like [XEP-0227](https://xmpp.org/extensions/xep-0227.html) does for servers and write that up in a new XEP.
> 
> Then implement that new XEP in commonly used clients like Conversations, Gajim or Monal.
> 
> That's nothing I will do, but I'm happy to take pull requests.

well, conversations/quicksy are able to do so.. maybe to use their functioning system?

**Comment by tmolitor-stud-tu** [comment]
> well, conversations/quicksy are able to do so.. maybe to use their functioning system?

No, different database format, different programming language and even different operating system.

In my opinion, iCloud backups are good enough. If, one day, there is no other issue in this issue tracker left, I might implement a second backup, but not any sooner: there are way more important features and bugfixes to implement.
